app/views/accounts/show.html.slim

```diff
@@ -45,6 +45,12 @@ h1.hide = t('titles.account')
     content: content_tag(:em, @view_model.totp_content),
     action: @view_model.totp_partial
 
+  - if FeatureManagement.webauthn_enabled?
+    = render 'account_item',
```

Comment:
This is fine for now. The reason the ticket for listing configurations was a dependency for this work was to ensure that we're aware of the way listing a number of configurations changes how this section of the accounts page works. It will become more obvious when that ticket moves into in-progress.